### ISSUE TITLE
Reproduce the results

### ISSUE THREAD

**Issue by Littleyezi** [automl/nas_benchmarks#22]
Hi, Thank you for this work! I would like to ask if I want to reproduce the paper in the paper, do I run the code in experiments_script directly, and the json file obtained is the result described in the paper? Are there any details that I need to pay attention to or need to modify?Looking forward to your reply.

**Comment by aaronkl** [comment]
Yes, you just need to run the scripts in the experiment_script folder. In the paper we ran each optimizer multiple times (~100 times IIRC) and the figures show the averaged results of all the json files

**Comment by Littleyezi** [comment]
Thanks for all.😊
